### DISCUSSION TITLE
code 500

### DISCUSSION THREAD

**Discussion by PeteTheWizYT** [supabase/supabase#28184] [Questions]
anybody? 

{
  "code": 500,
  "message": "An error has occurred: Failed to fetch",
  "requestId": "cb2e69c6-9b01-486f-9bc7-d741dec22ecd"
}

**Comment by GaryAustin1** [comment]
Check your Gateway API and Postgres logs to see if there is more info on any error and that the call gets to Supabase.
Failed to Fetch is one of the most generic errors and may not be coming from Supabase itself.

You will need to give more info on your environment, hosting/self-hosted/CLI, the call that is failing.

**Reply by PeteTheWizYT** [reply]
this is in Supha base, flutterflow, first set-up (tuttorial flutterflow)

**Reply by PeteTheWizYT** [reply]
compleeeeet NOOB (sorry)

**Reply by GaryAustin1** [reply]
You will need to show some code and narrow down what is getting the 500 error.  Did you look at the Dashboard UI logs as mentioned?

You might be better off trying Supabase Discord also.  It is more interactive than here and seems easier to go back and forth with someone working thru a bug.

I personally can't help if it is a Flutter issue, versus tracking a Supabase bug, but with the info you have presented so far not sure anyone can help you.

**Reply by PeteTheWizYT** [reply]
<img width="445" alt="image" src="https://github.com/user-attachments/assets/d99b549c-ffbf-437d-89e3-f18de4fb6795">
<img width="445" alt="image" src="https://github.com/user-attachments/assets/1232c30a-c41e-4c03-b14a-ea4a3aa21d43">
<img width="445" alt="image" src="https://github.com/user-attachments/assets/5e2f7015-7425-4a56-8bea-f9f9c0ef7fbd">
<img width="445" alt="image" src="https://github.com/user-attachments/assets/006a5784-4dc5-4d10-991d-c74bc5b4b3da">
this is what is see in my supha (discordlink?)

**Reply by PeteTheWizYT** [reply]
no other messages, no other logs, nothing

**Reply by PeteTheWizYT** [reply]
and in flutter no bugs or nothing to see

**Reply by PeteTheWizYT** [reply]
data base safe error new user (when i try to make an account, so is the back end code from that, or is the front code from that...) Thanks ill keep looking

**Reply by GaryAustin1** [reply]
OK if you are getting a database error when creating a new user then the Postgres logs in the dashboard should show you the error.   Usually if that is the error, you have a trigger function on auth.users that is failing because of bad coding or column names/types etc.  Many of the guides have such a trigger function for creating a user management profile table.
